ldk/javascript/src/types/oliveHelps/index.d.ts

```diff
@@ -268,28 +271,28 @@ declare namespace OliveHelps {
   type Button = Component<WhisperComponentType.Button> & {
     buttonStyle?: ButtonStyle;
     label: string;
-    onClick: () => void;
+    onClick: (whisper: Whisper) => void;
```

Comment:
Starting to think that we should introduce a new type for these Whisper onEvent handlers... not for immediate action right now, just thinking out loud.